cluster/gce/util.sh

```diff
@@ -783,6 +783,35 @@ function teardown-monitoring {
   fi
 }
 
+function setup-fluentd-elasticsearch-logging {
+  echo .. setting up Fluentd to Elasticsearch logging with Kibana viewer
+  local -r kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+  if "${kubectl}" create -f "${KUBE_ROOT}"/cluster/addons/fluentd-elasticsearch/es-pod.yml &> /dev/null && \
```

Comment:
I agree - @vishh should change his json to yaml.  I like .yaml, fwiw, and
it is more prevalent in our tree

On Wed, Jan 7, 2015 at 4:20 PM, Satnam Singh <notifications@github.com>
wrote:

> In cluster/gce/util.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3292#discussion-diff-22627653>
> :
>
> > @@ -783,6 +783,35 @@ function teardown-monitoring {
> >    fi
> >  }
> >
> > +function setup-fluentd-elasticsearch-logging {
> > +  echo .. setting up Fluentd to Elasticsearch logging with Kibana viewer
> > +  local -r kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
> > +  if "${kubectl}" create -f "${KUBE_ROOT}"/cluster/addons/fluentd-elasticsearch/es-pod.yml &> /dev/null && \
>
> When I wrote my first YAML file for this project I tried to fit in with
> what others had written and used ".yml" even though my fingers were
> programmed to type ".yaml". I am happy to change the suffixes if someone
> declares the standard. For human written files designed to be read by
> humans YAML seems like a much better choice than JSON.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3292/files#r22627653>
> .
>